cmd/main.go

```diff
@@ -0,0 +1,20 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+Copyright Contributors to the Submariner project.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import "github.com/submariner-io/submariner-operator/cmd/subctl"
+
+func main() {
```

Comment:
Do we need a `main.go` in `cmd` itself, or in `cmd/subctl`? (Assuming your goal is still `go install github.com/submariner-io/submariner-operator/cmd/subctl`.)

Reply:
yes. `main.go` initialises `subctl ` package under `cmd/subctl` and is generating by `cobra init --pkg-name subctl`